### DISCUSSION TITLE
How to identify crescendo and decrescendo in musicXML

### DISCUSSION THREAD

**Discussion by bajie-code** [CoderLine/alphaTab#1726] [Q&A]
How to identify crescendo and decrescendo in musicXML？
Example：
```xml
<direction-type>
 <wedge default-x="11" default-y="-77" color="#000000" type="crescendo" />
</direction-type>
```

**Comment by Danielku15** [comment]
I am not sure if I understand your question. This repository is about alphaTab. You can learn more about MusicXML here: https://www.musicxml.com/

If you have some questions that relates to alphaTab, please try to rephrase to make it more clear what you need.
